src/Model/Contact/User.php

```diff
@@ -105,6 +109,10 @@ public static function updateByContactUpdate(array $fields, array $condition)
 			$fields['ignored'] = $fields['readonly'];
 		}
 
+		if (!empty($fields['self'])) {
+			$fields['rel'] = Contact::SELF;
```

Comment:
Do we need to store any action on our own contact?

Reply:
I don't understand the question. We do have a "self" contact. And by now I think that we need to transform this to a structure with only a public contact and a "user-contact" table as well.

Reply:
I forgot that we indeed need to have the relationship between the user and their own contact.